lib/dataset-tools/splitter.js

```diff
@@ -53,7 +53,7 @@ const SPLIT_STRATEGIES = {
     program: (id, sentence, program) => join(requoteProgram(program)),
     combination: (id, sentence, program) => {
         const functions = Array.from(getFunctions(program));
-        if (functions.length <= 1)
+        if (functions.length === 0)
```

Comment:
getFunctions was being used here, and with the new getFunctions behavior, I think this change was necessary. But I would appreciate an extra pair of eyes, as I'm not familiar with the  rationale in this decision for train - eval split.

Reply:
No, this is not correct. getFunctions() returns the list of functions, and this function checks for one or two functions.
That fact that getFunctions() also returned the dialogue act was _the_ bug: it was never meant to do that. And indeed, it did not do that for single-shot dataset, because single-shot datasets don't have a dialogue act. So this code was correct as it was.

Reply:
That makes sense. Reverting